python/GafferTest/TestCase.py

```diff
@@ -299,7 +299,7 @@ def checkPlugs( graphComponent ) :
 	# Use `nodesToIgnore` with caution : the only good reason for using it is to
 	# ignore compatibility stubs used to load old nodes and convert them into new
 	# ones.
-	def assertNodesConstructWithDefaultValues( self, module, nodesToIgnore = None ) :
+	def assertNodesConstructWithDefaultValues( self, module, nodesToIgnore = None, plugsToIgnore = None ) :
```

Comment:
Is there any _good_ reason to use `plugsToIgnore` ? By which I mean one which doesn't mask a serialisation bug? I'm not completely averse to adding it - it's better to test everything with a couple of exceptions than not to test at all. But unless we think there's a valid reason to use it (I don't think there is for the native Gaffer codebase) then we should document that.